From 68a9b143d02436729e024f081c8d77b42f2e9d7f Mon Sep 17 00:00:00 2001
From: Tom Finegan <tomfinegan@google.com>
Date: Wed, 13 Jun 2018 09:39:28 -0700
Subject: [PATCH] Fix avx512 related MSVC build failure.

Check GCC specific AVX512 flags only when GCC is enabled.

Change-Id: I15dc2a0dbf8bce37f4364fedfd34a0a34882104b
---
 build/make/configure.sh | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/build/make/configure.sh b/build/make/configure.sh
index e0e9fd1a73..37b410edb6 100644
--- a/build/make/configure.sh
+++ b/build/make/configure.sh
@@ -1335,13 +1335,15 @@ EOF
             check_gcc_machine_options $ext avx512f avx512cd avx512bw avx512dq avx512vl
 
             # Confirm that the compiler really supports avx512.
-            check_cc -mavx512f <<EOF || RTCD_OPTIONS="${RTCD_OPTIONS}--disable-avx512 " && soft_disable avx512
+            if enabled gcc; then
+              check_cc -mavx512f <<EOF || RTCD_OPTIONS="${RTCD_OPTIONS}--disable-avx512 " && soft_disable avx512
 #include <immintrin.h>
 void f(void) {
   __m512i x = _mm512_set1_epi16(0);
   (void)x;
 }
 EOF
+            fi
           else
             # use the shortened version for the flag: sse4_1 -> sse4
             check_gcc_machine_option ${ext%_*} $ext
